guidance/credential-sharing.md

```diff
@@ -1,27 +1,23 @@
 # Helpful Engineering Shared Credentials
 
-Whenever possible, no account should be shared with other users.  However, in some instances (e.g. social media accounts, some email accounts, *et cetera*) it will be required to share a single set of credentials with multiple users using a tool called [LastPass](https://www.lastpass.com).
+Whenever possible, no account should be shared with other users. However, in some instances (e.g. social media accounts, some email accounts, *et cetera*) it will be required to share a single set of credentials with multiple users by using a tool called [1Password](https://1password.com).
 
- ⚠️ Credentials should **never** be shared outside LastPass: please **don't** send passwords in plaintext by any other means.
+The DevOps and Infosec teams maintain a master account to store, manage and share the credentials with the approved users. Thereby, passwords will be automatically synchronized with the other users' 1Password vaults when they are periodically rotated for security reasons.
```

Comment:
```suggestion
The DevOps and Infosec teams maintain a master account to store, manage and share the credentials with the approved users. Thereby, passwords will be automatically synchronized with the other users' 1Password vaults when they change.
```

Maybe we should make the sentence a bit more general.
_Note: this time, the `diff` is meaningful_